convert/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "extism-convert"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
 authors = ["The Extism Authors", "oss@extism.org"]
 license = "BSD-3-Clause"
```

I think to get it on crates.io we need to add:

```suggestion
license = "BSD-3-Clause"
readme = "./README.md"
```